package.json

```diff
@@ -19,12 +19,14 @@
     "@angular/platform-browser": "^15.0.0",
     "@angular/platform-browser-dynamic": "^15.0.0",
     "@angular/router": "^15.0.0",
+    "@angular/service-worker": "^15.0.0",
     "@capacitor/app": "4.1.1",
     "@capacitor/core": "4.7.0",
     "@capacitor/haptics": "4.1.0",
     "@capacitor/keyboard": "4.1.1",
     "@capacitor/status-bar": "4.1.1",
     "@ionic/angular": "^6.1.9",
+    "@ngx-pwa/local-storage": "^15.0.0",
```

Comment:
Do we also want to add avataaar while we're here?

Reply:
Let's hold off on that. I tested it and you can't customize the layout. Better to use the original JS library.

Reply:
Cool, sounds good. Go ahead and merge it